sway/desktop/output.c

```diff
@@ -755,8 +755,94 @@ static void render_container(struct sway_output *output,
 		render_container_tabbed(output, damage, con, parent_focused);
 		break;
 	case L_FLOATING:
-		// TODO
-		break;
+		sway_assert(false, "Didn't expect to see floating here");
+	}
+}
+
+static bool floater_intersects_output(struct sway_container *floater,
```

Comment:
This isn't enough, because popups/subsurfaces can be outside of this box and still need to get rendered.